### PR TITLE
Tegan's edits to The value of Platform Engineering

### PR DESCRIPTION
**Pull request by TeganAli**

### CODE REVIEW

**Review by mcasperson — APPROVED** [review]
lgtm

File: _includes/pe_value.html [hunk 1]
```diff
@@ -114,49 +109,49 @@ <h2>Quantifying the Business Value</h2>
...
+<p><i>Emma Maslen, the author of the post above, included shareholder value. She did this to capture cases where businesses were
+    working towards efforts like environmental initiatives or diversity, equity, and inclusivity that may not necessarily
+    impact risk, cost, or revenue. In most cases, shareholder value is a consequence of changes to the other 3 pain
     points.</i></p>
 
 <p>Fortunately, an IDP has plenty of scope to impact business value positively. By infusing best practices at scale
-    throughout your DevOps teams, it is reasonable to argue that your IDP directly reduces friction, improves DevEx,
-    improves deployment frequency, or improves compliance. Importantly, these improvements are repeatable and scalable.
-    The final step is to describe these improvements in terms of business concerns like risk, cost, or revenue.</p>
+    throughout your DevOps teams, it is reasonable to argue that your IDP directly:
```
Comment (mcasperson) [inline _includes/pe_value.html:134]: This needs a closing </p> tag

**Review by mcasperson — CHANGES_REQUESTED** [review]
The validation check has run and this PR introduces some invalid HTML. This will need to be fixed before we can merge the changes.

File: _includes/pe_value.html (hunk 1 above)
Comment (TeganAli) [inline _includes/pe_value.html:134]:
```suggestion
    throughout your DevOps teams, it is reasonable to argue that your IDP directly: </p>
```

Comment (TeganAli) [inline _includes/pe_value.html]:
```suggestion
    throughout your DevOps teams, it is reasonable to argue that your IDP directly:</p>
```

File: _includes/pe_value.html [hunk 2]
```diff
@@ -114,49 +109,49 @@ <h2>Quantifying the Business Value</h2>
...
+<p><i>Emma Maslen, the author of the post above, included shareholder value. She did this to capture cases where businesses were
+    working towards efforts like environmental initiatives or diversity, equity, and inclusivity that may not necessarily
+    impact risk, cost, or revenue. In most cases, shareholder value is a consequence of changes to the other 3 pain
     points.</i></p>
 
 <p>Fortunately, an IDP has plenty of scope to impact business value positively. By infusing best practices at scale
-    throughout your DevOps teams, it is reasonable to argue that your IDP directly reduces friction, improves DevEx,
-    improves deployment frequency, or improves compliance. Importantly, these improvements are repeatable and scalable.
-    The final step is to describe these improvements in terms of business concerns like risk, cost, or revenue.</p>
+    throughout your DevOps teams, it is reasonable to argue that your IDP directly: </p>
```
Comment (TeganAli) [inline _includes/pe_value.html:134]:
```suggestion
    throughout your DevOps teams, it is reasonable to argue that your IDP directly:</p>
```

**Comment by TeganAli** [comment]
Hey @mcasperson, I can't work out how to fix the validation issues here, but wondering if I should just copy all my changes over after I create a branch, which might be easier for me, then delete this?

File: _includes/pe_value.html [hunk 3]
```diff
@@ -75,22 +72,20 @@ <h2>Platform Engineering by the Numbers</h2>
     </li>
 </ol>
 
-<p>While the ASBX team doesn't specifically describe itself as a platform team, a focus on "undifferentiated work
+<p>While the ASBX team doesn't specifically describe itself as a platform team, they focus on "undifferentiated work
```
Comment (TeganAli) [inline _includes/pe_value.html:75]:
```suggestion
<p>While the ASBX team doesn't specifically describe itself as a platform team, a focus on "undifferentiated work
```

File: _includes/pe_value.html [hunk 4]
```diff
@@ -114,49 +109,49 @@ <h2>Quantifying the Business Value</h2>
...
+<p>The post “Identified Pain Is About Customer Outcomes” (<a href="https://oc.to/iktukN">https://oc.to/iktukN</a>) lists 4 concerns, or pain points,
     that all business initiatives must consider:</p>
 
 <ul>
     <li>Revenue – is there a pain impacting their booking revenue? Does your solution solve this pain?</li>
     <li>Cost – is there a pain driving up the cost of doing business? Can your solution solve this pain?</li>
     <li>Risk – is there a new regulation they are in danger of breaching? Does your solution mitigate the pain?</li>
-    <li>Shareholder Commitments – have they committed to something to their shareholders, which without your solution
-        they will fail to deliver?
+    <li>Shareholder commitments – have they committed to something to their shareholders, which they'll fail to deliver without your solution?
```
Comment (TeganAli) [inline _includes/pe_value.html:119]:
```suggestion
    <li>Shareholder commitments – have they committed to something to their shareholders, which they'll fail to deliver without your solution?</li>
```

File: _includes/pe_value.html [hunk 5]
```diff
@@ -114,49 +109,49 @@ <h2>Quantifying the Business Value</h2>
...
     that all business initiatives must consider:</p>
 
 <ul>
     <li>Revenue – is there a pain impacting their booking revenue? Does your solution solve this pain?</li>
     <li>Cost – is there a pain driving up the cost of doing business? Can your solution solve this pain?</li>
     <li>Risk – is there a new regulation they are in danger of breaching? Does your solution mitigate the pain?</li>
-    <li>Shareholder Commitments – have they committed to something to their shareholders, which without your solution
-        they will fail to deliver?
+    <li>Shareholder commitments – have they committed to something to their shareholders, which they'll fail to deliver without your solution?</li>
     </li>
```
Comment (TeganAli) [inline _includes/pe_value.html:120]:
```suggestion

```

**Review by mcasperson — APPROVED** [review]
The HTML validation check has passed, so this is good to go.

**Pull request merged**